### PR TITLE
Added support for imagelayers in Tiled loader, closes #155

### LINKED ISSUE #155: Add a "TiledImage.hx" to the Tiled library
Similar to this: 
https://github.com/kasoki/openfl-tiled-flixel/search?utf8=%E2%9C%93&q=ImageLayer&type=Code
https://github.com/kasoki/openfl-tiled/blob/fc82b84c7c49a5ec80a86cc5c62566542a8deb14/openfl/tiled/ImageLayer.hx

Right now (I think) the flixel-addons Tiled classes do not support Tiled's image layers, though such support can be very useful (for example, you can use Tiled as a non-tiles level editor).

### PR DESCRIPTION
**Pull request by sebbernery**
Fix issue #155 
Add support for imagelayer of Tiled ( here : http://doc.mapeditor.org/reference/tmx-map-format/ <imagelayer>)

### CODE REVIEW

**Comment by Gama11** [comment]
Thanks!

Would be nice if this was used in the tiled demo somewhere, both to test it works correctly (now and in the future) and to show off the usage.

File: flixel/addons/editors/tiled/TiledImageLayer.hx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package flixel.addons.editors.tiled;
+
+import haxe.xml.Fast;
+
+
+class TiledImageLayer extends TiledLayer
+{
+	public var image_path:String;
```
Comment (Gama11) [inline flixel/addons/editors/tiled/TiledImageLayer.hx]: Flixel uses camelCase, so this should be `imagePath`.

**Comment by sebbernery** [comment]
Thank you for your feedbacks. I modified the Tiled demo to add an image layer : https://github.com/HaxeFlixel/flixel-demos/pull/204 .

**Comment by sruloart** [comment]
Hooray! :)

**Pull request merged**